.vscode/settings.json

```diff
@@ -20,5 +20,9 @@
   "[json][jsonc]": {
     "editor.formatOnSave": true,
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
This isn't related to Storybook? Why do you need this? You should be able to use ESLInt and not the default VSCode formatter?